DOC - More fixes and improvements to the user guide (#1574)
skrub-data/skrub: doc/modules/tablereport/exploring_dataframes_interactively.rst

.. |TableReport| replace:: :class:`~skrub.TableReport`
.. |set_config| replace:: :func:`~skrub.set_config`
.. |column_associations| replace:: :func:`~skrub.column_associations`

.. _user_guide_table_report_start:

Exploring dataframes interactively with the |TableReport|
=========================================================

> The |TableReport| gives a high-level overview of the given dataframe, suitable for
> quick exploratory analysis of series and dataframes. The report shows the first

Reviewer: ```suggestion
The |TableReport| gives a high-level overview of a Dataframe or Series, suitable for
quick exploratory analysis. The report shows the first
```

> and last 5 rows of the dataframe (decided by the ``n_rows`` parameter), as well

Reviewer: an actual report would be useful here, and maybe void the need for a textual description

Author: is it possible to embed a report in the documentation?
